clash-prelude/src/Clash/Sized/RTree.hs

```diff
@@ -105,26 +112,26 @@ let populationCount' :: (KnownNat k, KnownNat (2^k)) => BitVector (2^k) -> Index
 -- * Only has elements at the leaf of the tree
 -- * A tree of depth /d/ has /2^d/ elements.
 data RTree :: Nat -> Type -> Type where
-  LR_ :: a -> RTree 0 a
-  BR_ :: RTree d a -> RTree d a -> RTree (d+1) a
+  RLeaf :: a -> RTree 0 a
+  RBranch :: RTree d a -> RTree d a -> RTree (d+1) a
```

Comment:
This warrants a `CHANGED` changelog entry, right?

Reply:
Mmm… I think it’d sooner be ADDED rather than CHANGED, since the constructors weren’t exported before.

Reply:
Yes, I'm on it :-)

Reply:
Ok, I have added this.